public/components/Notifications/NotificationForm.tsx

```diff
@@ -168,18 +185,27 @@ export const NotificationForm: React.FC<NotificationFormProps> = ({
                   />
                 </EuiCompressedFormRow>
               </EuiFlexItem>
-              {prepareMessage && (
+
+              <EuiFlexItem>
+                <EuiCompressedCheckbox
+                  id="notification-message-preview-checkbox"
+                  label="Preview message"
+                  checked={displayPreview}
+                  onChange={onDisplayPreviewChange}
+                />
+              </EuiFlexItem>
+              {displayPreview ? (
```

Comment:
Nit: We can also do `displayPreview && (<Comp />) instead of the ternary to keep it short

Reply:
Done